scripts/git/update_to_llvm_syncpoint.py

```diff
@@ -116,15 +118,17 @@ def main(args):
   print(f"  LLVM Bazel Path: {args.llvm_bazel_path}")
   print(f"  TensorFlow Path: {args.tensorflow_path}")
 
-  current_llvm_commit = get_commit(args.llvm_path)
+  current_llvm_commit = parse_rev(args.llvm_path, "HEAD")
+  current_llvm_bazel_commit = parse_rev(args.llvm_bazel_path, "HEAD")
+  current_tf_commit = parse_rev(args.tensorflow_path, "HEAD")
   print("Current Commits:")
   print(f"  llvm = {current_llvm_commit}")
-  print(f"  llvm_bazel = {get_commit(args.llvm_bazel_path)}")
-  print(f"  tensorflow = {get_commit(args.tensorflow_path)}")
+  print(f"  llvm_bazel = {current_llvm_bazel_commit}")
+  print(f"  tensorflow = {current_tf_commit}")
```

Comment:
Is there a reason to prefer creating and using variables here if they're only used once?

Reply:
Not a super strong reason. Without the variables things were getting a bit messy with the quoting and I thought it was hard to read.

Reply:
Ah, I didn't notice that `"HEAD"` was added – coffee's still kicking in :)